src/components/experiment-tracking/tooltip/tooltip.scss

```diff
@@ -0,0 +1,103 @@
+@use '../../../styles/variables' as colors;
+
+@mixin fade-in($waitTime) {
+  animation: wait #{$waitTime}, fade-in 800ms #{$waitTime};
+}
+
+@keyframes wait {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+$triangle-size: 10px;
+
+.tooltip {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  opacity: 0;
+  padding: 10px 30px 10px 10px;
+  position: absolute;
+}
+
+.tooltip--show {
+  @include fade-in('700ms');
+  animation-fill-mode: forwards;
+}
+
+.tooltip-arrow {
+  display: inline-block;
+  height: 0;
+  left: 1px;
+
+  margin-right: 1.6em;
+  margin-top: -1.2em;
+
+  position: absolute;
+  top: 22px;
+  white-space: nowrap;
+  width: 0;
+}
+
+.tooltip-arrow--right {
+  &::before {
+    border-left: $triangle-size solid transparent;
+    border-top: $triangle-size solid var(--color-bg-alt);
+
+    content: '';
+
+    height: 0;
+    left: -$triangle-size + 0.5;
+    position: absolute;
+    top: calc(50% - #{$triangle-size});
+    width: 0;
+  }
+}
+
+.tooltip-arrow--left {
+  &::before {
+    border-right: $triangle-size solid transparent;
+    border-top: $triangle-size solid var(--color-bg-alt);
+
+    content: '';
+
+    height: 0;
+    position: absolute;
+    right: -225.5px;
+    top: calc(50% - #{$triangle-size});
+    width: 0;
+  }
+}
+
+.tooltip-label,
+.tooltip-value {
+  font-size: 12px;
+  margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 180px;
+}
```

Comment:
It doesn't seem the font styles are matching with the designs. This is yours:

<img width="325" alt="image" src="https://user-images.githubusercontent.com/16869061/204525124-af746382-56f5-4743-a979-0cb6699c3af0.png">

And this is the design:

<img width="284" alt="image" src="https://user-images.githubusercontent.com/16869061/204525221-cb31d64a-1bb2-458c-933f-93b178f478ef.png">

Can we ensure they match?